pyerrors/fits.py

```diff
@@ -916,11 +730,18 @@ def chisqfunc_compact(d):
 
     output.fit_parameters = result
 
+    # Hotelling t-squared p-value for correlated fits.
     if kwargs.get('correlated_fit') is True:
         n_cov = np.min(np.vectorize(lambda x_all: x_all.N)(y_all))
         output.t2_p_value = 1 - scipy.stats.f.cdf((n_cov - output.dof) / (output.dof * (n_cov - 1)) * output.chisquare,
                                                   output.dof, n_cov - output.dof)
 
+    if kwargs.get('resplot') is True:
+        residual_plot(x, y, func, result)
+
+    if kwargs.get('qqplot') is True:
+        qqplot(x, y, func, result)
+
```

Comment:
This does not work if dictionaries are passed to `least_squares`. I guess `residual_plot` and `qqplot` have to be adapted to dictionaries, as well.

Reply:
Thanks for spotting this. My changes should make the two routines work with combined fits.